scipy/stats/_unuran/unuran_wrapper.pyx.templ

```diff
@@ -969,6 +969,185 @@ cdef class TransformedDensityRejection(Method):
         return np.asarray(out).reshape(oshape)[()]
 
 
+cdef class SimpleRatioUniforms(Method):
+    r"""
+    SimpleRatioUniforms(dist, *, mode=None, pdf_area=1, domain=None, cdf_at_mode=None, random_state=None)
+
+    Simple Ratio-of-Uniforms (SROU) Method.
+
+    SROU is based on the ratio-of-uniforms method that uses universal inequalities for constructing
+    a (universal) bounding rectangle. It works for T-concave distributions with T(x) = -1/sqrt(x).
+
+    Parameters
+    ----------
+    dist : object
+        An instance of a class with ``pdf`` method.
+
+        * ``pdf``: PDF of the distribution. The signature of the PDF is
+          expected to be: ``def pdf(self, x: float) -> float``. i.e.
+          the PDF should accept a Python float and
+          return a Python float. It doesn't need to integrate to 1 i.e.
+          the PDF doesn't need to be normalized.
```

Comment:
Then in this case the `pdf_area ` should be adapted right? If yes, we should note it here as well.

Reply:
> Then in this case the `pdf_area `should be adapted right? If yes, we should note it here as well

Yes. I will add a note.